packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -37,6 +37,8 @@ export interface RollupLwcOptions {
     enableScopedSlots?: boolean;
     /** The configuration to pass to `@lwc/compiler` to disable synthetic shadow support */
     disableSyntheticShadowSupport?: boolean;
+    /** Boolean to determine if LWS is on or off */
```

Comment:
Might be good to define Lightning Web Security here – we have a lot of acronyms. 😛